### PR TITLE
Added Crashlytics

### PR DESCRIPTION
**Pull request by jamesstout**
What does this implement/fix? Explain your changes.
---------------------------------------------------
Added Crashlytics

Does this close any currently open issues?
------------------------------------------
…

Any relevant logs, error output, etc?
-------------------------------------

Any other comments?
-------------------
…

Where has this been tested?
---------------------------
**Devices/Simulators:** iMac19,1

**macOS Version:** 10.15.7 (19H2)

**Sequel-Ace Version:** main-3_0_0

### CODE REVIEW

**Comment by Kaspik** [comment]
Please add `Pods` to the gitignore first!

File: Source/Controllers/SPAppController.h [hunk 1]
```diff
@@ -35,6 +35,8 @@
 @class SPBundleEditorController;
 @class SPWindowController;
 
+@import Firebase;
```
Comment (Kaspik) [inline Source/Controllers/SPAppController.h:38]: Can be only in `m` file?

File: Resources/Pods-Sequel Ace-acknowledgements.markdown [hunk 2]
```diff
@@ -0,0 +1,1748 @@
+# Acknowledgements
+This application makes use of the following third party libraries:
+
+## Firebase
```
Comment (Kaspik) [inline Resources/Pods-Sequel Ace-acknowledgements.markdown:4]: 👍

File: Source/Controllers/SPAppController.h (hunk 1 above)
Comment (jamesstout) [reply Source/Controllers/SPAppController.h:38]: I couldn't get it to work in an .h file. I tried. Needs modules, then get errors. It's a shame as I wanted an app-wide available macro. Maybe you know more about this?

**Comment by jamesstout** [comment]
> Please remove Pods folder, commit, then add `Pods` to the gitignore first, commit, and then install it again :)

The repo will still be contaminated. Would you prefer I created a new branch and redo so the main repo isn't contaminated.

Or use BFG (I'd rather not)

**Comment by Jason-Morcos** [comment]
> > Please remove Pods folder, commit, then add `Pods` to the gitignore first, commit, and then install it again :)
> 
> The repo will still be contaminated. Would you prefer I created a new branch and redo so the main repo isn't contaminated.
> 
> Or use BFG (I'd rather not)

Yeah I would create a new branch and do that. I pulled some firebase pods in before in another project without git ignoring first and regret it to this day. I think the start a new branch and just redo the minimal changes by hand approach is probably cleanest.

**Comment by Jason-Morcos** [comment]
@Kaspik We may need to tweak the build script for tests (like the automated GitHub ones) to install cocoa pods if we gitignore the pods dir. Is that problematic at all?

**Comment by jamesstout** [comment]
> > > Please remove Pods folder, commit, then add `Pods` to the gitignore first, commit, and then install it again :)
> > 
> > 
> > The repo will still be contaminated. Would you prefer I created a new branch and redo so the main repo isn't contaminated.
> > Or use BFG (I'd rather not)
> 
> Yeah I would create a new branch and do that. I pulled some firebase pods in before in another project without git ignoring first and regret it to this day. I think the start a new branch and just redo the minimal changes by hand approach is probably cleanest.

Done.

**Comment by jamesstout** [comment]
> @Kaspik We may need to tweak the build script for tests (like the automated GitHub ones) to install cocoa pods if we gitignore the pods dir. Is that problematic at all?

Ah yes ... builds are failing I think.

**Comment by jamesstout** [comment]
Closing. Replaced by #432

**Pull request closed without merging**